atd-vzd/triggers/update_cr3_location.sql

```diff
@@ -0,0 +1,29 @@
+-- This trigger function updates the location_id of a record in atd_txdot_crashes on insert or update.
+CREATE OR REPLACE FUNCTION update_cr3_location()
+    RETURNS TRIGGER AS $$
+        BEGIN
+            -- If crash is level 5 try to associate crash to level 5 polygon and return the location id
```

Comment:
Sorry to dig this back up, but are we sure that the VZ team is on board with using the current level 5 polygons? Or, we might be holding off on this until the new level 5s are ready for production.

Reply:
This is what Xavier wanted me to change this code to in our last meeting soo i think so :shrug: